src/main/java/seedu/address/logic/commands/ViewAvailableCommand.java

```diff
@@ -0,0 +1,92 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.time.LocalDate;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.function.Predicate;
+
+import javafx.collections.ObservableList;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.model.Model;
+import seedu.address.model.appointment.Appointment;
+import seedu.address.model.timeslots.Timeslots;
+
+/**
+ * A ViewAvailableCommand instance which user can use to view available timeslots on a specified date
+ */
+public class ViewAvailableCommand extends Command {
+    public static final String COMMAND_WORD = "view";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + " /on DATE";
```

Comment:
Update error message to give more information about view command.
Eg.
Invalid command format! 
find: Finds all patients whose names contain any of the specified keywords (case-insensitive) and displays them as a list with index numbers.
Parameters: KEYWORD [MORE_KEYWORDS]...
Example: find alice bob charlie

Reply:
Ok sir! done